packages/website/blog/2024-04-30-announcing-typescript-eslint-v8-beta.md

```diff
@@ -0,0 +1,226 @@
+
+### ⏳ ESLint v9 Support
+
+typescript-eslint v8 ships will full support for ESLint v9.
+
+typescript-eslint v7 was our first version that supported ESLint's [new "flat" config file format](https://eslint.org/docs/latest/use/configure/configuration-files), which was already available in ESLint v8.
+ESLint v9 still supports ESLint's [older legacy config file format](https://eslint.org/docs/latest/use/configure/configuration-files-deprecated) so our tooling does as well.
+However, ESLint v9 also includes a set of breaking changes that we added support for in typescript-eslint v8.
+See the [ESLint v9 release blog post](https://eslint.org/blog/2024/04/eslint-v9.0.0-released) and for more details.
+
+### ⏳ Project Service
+
+The biggest new feature added in this version is the stability of our new "project service".
+In short, the project service is a new way to enable [typed linting](/getting-started/typed-linting) that is generally _easier to configure_ and _faster at runtime_ than our previous offerings.
+It's been experimentally available since v6.1.0 under the name `EXPERIMENTAL_useProjectService`; now, we've renamed it to `projectService`.
+
+You can use the new project service in your configuration instead of the previous `parserOptions.project`:
+
+```js title="eslint.config.js"
+import eslint from '@eslint/js';
+import tseslint from 'typescript-eslint';
+
+export default tseslint.config(
+  eslint.configs.recommended,
+  ...tseslint.configs.recommendedTypeChecked,
+  {
+    languageOptions: {
+      parserOptions: {
+        // Remove this line
+        project: true,
+        // Add this line
+        projectService: true,
+        tsconfigRootDir: import.meta.dirname,
+      },
+    },
+  },
+);
+```
+
+The project service will automatically find the closest `tsconfig.json` for each file (like `project: true`).
+It also allows enabling typed linting for files not explicitly included in a `tsconfig.json`.
+This should remove the need for custom `tsconfig.eslint.json` files to lint files like `eslint.config.js`!
+
+```js title="eslint.config.js"
+import eslint from '@eslint/js';
+import tseslint from 'typescript-eslint';
+
+export default tseslint.config(
+  eslint.configs.recommended,
+  ...tseslint.configs.recommendedTypeChecked,
+  {
+    languageOptions: {
+      parserOptions: {
+        // Remove this line
+        project: ['packages/*/tsconfig.json', 'tsconfig.eslint.json'],
+        // Add this line
+        projectService: {
+          allowDefaultProjectForFiles: ['./*.js'],
+          defaultProject: './tsconfig.json',
+        },
+        tsconfigRootDir: import.meta.dirname,
+      },
+    },
+  },
+);
+```
+
+Internally, the project service uses the same TypeScript APIs that editors such as VS Code use.
+Doing so should make it harder to accidentally configure different type information for ESLint than what you see in day-to-day editing.
+
+We're thrilled to have this option promoted to stable in v8.
```

Comment:
(nit) Consider throwing in an exclamation point on one of these sentences to show how thrilled we are! Or maybe put a "Stay tuned!" at the end?

Reply:
🚀